benchmarks/automated/README.md

```diff
@@ -17,10 +17,10 @@ Check out a sample vgg11 model config at the path: `tests/suite/vgg11.yaml`
 ```
```

Comment:
@nskool please change AmazonIAMFullAccess to IAMFullAccess

Reply:
Fixed, thanks